src/client/cypress/README.md

```diff
@@ -0,0 +1,56 @@
+[![GitHub Workflow Status](https://img.shields.io/github/workflow/status/AdaptiveConsulting/ReactiveTraderCloud/CI)](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/actions?query=workflow%3ACI)
+[![GitHub release (latest by date)](https://img.shields.io/github/v/release/AdaptiveConsulting/ReactiveTraderCloud)](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/releases/latest)
+[![GitHub](https://img.shields.io/github/license/AdaptiveConsulting/ReactiveTraderCloud)](https://opensource.org/licenses/Apache-2.0)
+
+[![image](https://raw.githubusercontent.com/AdaptiveConsulting/ReactiveTrader/master/images/adaptive-logo.png)](http://weareadaptive.com/)
+
+# Reactive Trader®
+
+Reactive Trader® is a real-time FX trading platform designed to showcase reactive programming principles across the full application stack.
+
+Originally [written in WPF and .Net](https://github.com/AdaptiveConsulting/ReactiveTrader), and now in React/Redux, .Net and Node.js, we continue to evolve the platform to use the latest technologies.
+
+Please see [our Showcase page](https://weareadaptive.com/showcase/) for a full list of the latest features.
+
+![image](docs/reactive-trader.gif)
+
+
+## Setting up the Test
+
+
+1. Fork and clone the ReactiveTraderCloud repo
+
+2. Navigate to `/src/client` and install Cypress ([from the Cypress website](https://docs.cypress.io/guides/getting-started/installing-cypress))
+
+3. Start the application on localhost by hitting `npm run start`
+
+4. Open a browser and navigate to http://localhost to see the application running. 
+
+5. Navigate to `/src/client` to see the list of feature files
```

Comment:
This path is not correct.
Also, I think we should move everything out of client and into `src/cypress` or maybe `src/e2e` in case we ever move to a different framework.

Reply:
fixinf the comment in new commit